src/support/bytemuck.rs

```diff
@@ -0,0 +1,66 @@
+//! Support for the [`bytemuck`](https://crates.io/crates/bytemuck) crate.
+#![cfg(feature = "bytemuck")]
+#![cfg_attr(docsrs, doc(cfg(feature = "bytemuck")))]
+
+use bytemuck::Pod;
+use ruint::Uint;
+
+// Implement the `Pod` trait for `Uint` types with a size that is a multiple of 64, up to 1024.
+macro_rules! impl_pod {
+    ($($bits:expr),+ $(,)?) => {
+        $(
+            unsafe impl<const LIMBS: usize> bytemuck::Zeroable for Uint<{$bits}, LIMBS> {}
+            unsafe impl<const LIMBS: usize> Pod for Uint<{$bits}, LIMBS> where
+                [(); {$bits % 64}]: ,
```

Comment:
What does this clause do?

Reply:
Originally, I was trying to find a way to enforce a compile time check to ensure that `bits` was a multiple of 64, however this is not achieving that and I have simplified to the following:

```rust
macro_rules! impl_pod {
    ($(($bits:expr, $limbs:expr)),+ $(,)?) => {
        $(
            unsafe impl Pod for Uint<{$bits}, $limbs> {}
        )+
    };
}
```